### DISCUSSION TITLE
Themes

### DISCUSSION THREAD

**Discussion by otalado** [wavelog/wavelog#640] [Q&A] [answered]
I have tested the Cosmo, Cosmo Wide, Default, and Default Wide themes on Wavelog and noticed that all of them have a dark appearance, even if they are not labeled as such. I don't intend to start a debate about personal preferences, as I believe usability is more important than visual appearance in software.

However, it's important to consider that in any large group of users, there will be some individuals with vision problems. For example, I personally struggle to read white text on a black background, and I know others in my age group who have the same issue.

Therefore, I would like to request the inclusion of at least one light theme in Wavelog, such as a simple black-on-white theme or a theme similar to what is often called the Solarized Light theme, which I find more suitable for reading.

**Comment by HB9HIL** [comment, marked as answer]
Hello @otalado 

There has to be some kind of issue on your side, because **Cosmo and Default are light theme**. 

To change your theme, go to your User Account, change the theme (Wavelog Preferences) and click save down below. 

Cosmo Wide:

<img width="1920" alt="image" src="https://github.com/user-attachments/assets/b2468724-4c61-49b0-9553-3609b0f863b2">
---


Default Wide: 

<img width="1919" alt="image" src="https://github.com/user-attachments/assets/24f763d2-fa12-40a2-87da-bf8950bde8e0">
---


Cyborg Wide:

<img width="1920" alt="image" src="https://github.com/user-attachments/assets/4ccca0a5-0827-4ed5-9a07-8083990cc01c">
---


Darkly Wide: 

<img width="1920" alt="image" src="https://github.com/user-attachments/assets/54962e15-d83d-4a7f-9430-477c6470bcae">
---


Superhero Wide:

<img width="1920" alt="image" src="https://github.com/user-attachments/assets/95b01b3d-da07-499b-93c8-f831d30e93ab">

**Comment by otalado** [comment]
I have to apologize. I tried to change the theme through Admin -> Global Options as I did not go through the User Account settings yet. Typical newbie user screwup, sorry :(

**Comment by HB9HIL** [comment]
no Problem 😄 The global options -> appearance control for example the login screen or public log view
